Marlin/src/pins/pins.h

```diff
@@ -224,6 +224,8 @@
   #include "ramps/pins_RAMPS_CREALITY.h"            // ATmega2560                           env:mega2560
 #elif MB(DAGOMA_F5)
   #include "ramps/pins_DAGOMA_F5.h"                 // ATmega2560                           env:mega2560
+#elif MB(DAGOMA_D6)
+  #include "ramps/pins_DAGOMA_D6.h"                 // ATmega2560                           env:FYSETC_F6
```

Comment:
This should be env:mega2560ext instead of FYSETC_F6

Reply:
FYSETC_F6 was what Dagoma used. But I will change this once I am up and moving.